paypal-commercetools-extension/src/utils/map.utils.ts

```diff
@@ -174,6 +175,26 @@ export const mapCommercetoolsLineItemsToPayPalItems = (
   } as Item;
 };
 
+export const mapValidCommercetoolsLineItemsToPayPalItems = (
+  matchingAmounts: boolean,
+  isShipped: boolean,
+  lineItems?: LineItem[],
+  locale?: string
+) => {
+  if (!matchingAmounts || !lineItems) {
+    return null;
+  }
+  const relevantLineItems = lineItems.filter(
```

Comment:
I thought that you would actually keep the GiftLineItems but change the unit_amount to 0

Reply:
Tried - PayPal returns error if quantity is 0, also for prices there is a specification in the api that it should be >0, not >0